src/game.js

```diff
@@ -0,0 +1,41 @@
+import getRandomSingleDigit from './random.js';
+import getElements from './elements.js';
+
+function findWinners(_cars) {
+  const totalDistances = _cars.map((car) => car.distance);
+  const maxDistance = Math.max(...totalDistances);
+  return _cars
+    .filter((car) => {
+      return car.distance === maxDistance;
+    })
+    .map((winner) => winner.name);
```

Comment:
return 을 축약해서 더 간결하고 깔끔하게 작성할 수 있을거 같습니다!

```suggestion
  return _cars
    .filter((car) => car.distance === maxDistance)
    .map((winner) => winner.name);
```